backend/init_data/skills/dingtalk-docs/provider.py

```
@@ -0,0 +1,711 @@
+                        "knowledge_base_id": knowledge_base_id,
+                        "doc_title": doc_title,
+                        "attachment_id": attachment_id,
+                        "trigger_indexing": trigger_indexing,
+                        "trigger_summary": trigger_summary,
+                    },
+                },
+                "id": 1,
+            }
+
+            # Write payload to a temporary file to avoid shell injection
+            payload_file = "/tmp/mcp_payload.json"
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/mcp_payload.json"

(S108)